### PR TITLE
Modified readme file

### PR DESCRIPTION
**Pull request by lakshminarasimmanv**
This commit makes changes to the readme file including making links and texts bolder, formating, and making it uniform.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,8 +91,8 @@ along existing investments, codebases, and developer populations. There are a
 few languages that have followed this model for other ecosystems, and Carbon
 aims to fill an analogous role for C++:
 
--   JavaScript → TypeScript
--   Java → Kotlin
+-   JavaScript → **_TypeScript_**
+-   Java → **_Kotlin_**
```
Comment (geoffromer) [inline README.md:95]: I think the intent here is to put "Carbon" in boldface to emphasize that it's the part this project is adding. Making the other languages boldface would defeat the purpose.

File: README.md [hunk 2]
```diff
@@ -198,7 +198,7 @@ https://drive.google.com/drive/folders/1-rsUjiya7dSZ87L8kpZmu3MZghRVxzLA
 </a>
 
 Read more about
-[C++ interop in Carbon](/docs/design/interoperability/philosophy_and_goals.md).
+**[C++ interop in Carbon](/docs/design/interoperability/philosophy_and_goals.md)**.
```
Comment (geoffromer) [inline README.md:201]: What's the purpose of making this boldface?

File: README.md [hunk 3]
```diff
@@ -264,7 +264,8 @@ semantics onto C++ such as Rust-inspired
 You can get started playing with Carbon by checking out the codebase and using
 the Carbon explorer:
 
-```shell
+```
+shell
```
Comment (geoffromer) [inline README.md:268]: This was intentionally on the same like as the quotes, and doesn't work if it's on a separate line. See [here](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting) for more information.

Comment (sadikkuzu) [reply README.md:268]: I agree with @geoffromer 

@lakshminarasimmanv you can see the result of this difference:
https://github.com/lakshminarasimmanv/carbon-lang/tree/patch-1#getting-started
image

**Comment by lakshminarasimmanv** [comment]
thank you for the feedback.

**Pull request closed without merging**